build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
   id "org.sonarqube" version "3.1.1"
-  id "com.diffplug.spotless" version "5.12.2"
+  id "com.diffplug.spotless" version "5.12.4"
```

Comment:
This was not me.

Reply:
Yeah it seems to be coming from @dependabot in a7e6a56.